src/components/Editor/EditorMenu.js

```diff
@@ -75,14 +78,18 @@ function getMenuItems(
     event
   );
 
-  const pairedType = isOriginalId(selectedLocation.sourceId)
-    ? L10N.getStr("generated")
-    : L10N.getStr("original");
+  const isGenerated = isOriginalId(selectedLocation.sourceId);
+  const isPrettyTabOpen =
+    !selectedSource.get("isPrettyPrinted") &&
+    tabs.includes(getPrettySourceURL(selectedSource.get("url")));
 
   const jumpLabel = {
     accesskey: L10N.getStr("editor.jumpToMappedLocation1.accesskey"),
-    disabled: false,
-    label: L10N.getFormatStr("editor.jumpToMappedLocation1", pairedType),
+    disabled: isGenerated ? false : !isPrettyTabOpen,
```

Comment:
I'm a bit confused by this so i'll share the spec as i see it:

- original files can not be pretty printed
- generated files that have a source map cannot be pretty printed
- if the source has been pretty printed, jumping should jump between the pretty and "ugly" file :) this might be new behavior...